doc/source/security.rst

```diff
@@ -0,0 +1,90 @@
+.. _security:
+
+Security Considerations
+=======================
+
+Setting up HTTPS
+----------------
+
+Enabling HTTPS is an important part of keeping the internet secure for
+your users & the world at large. Zero to JupyterHub makes doing so quite
+easy since version 0.5, integrating with `Let's Encrypt <https://letsencrypt.org/>`_
+for free HTTPS certificates.
+
+You can also purchase your own SSL certificates from a certificate provider.
+
+1. Buy a domain name from a registrar. Pick whichever one you want.
```

Comment:
These steps seem a bit confusing to me.

It may be better to pull out the first 3 steps and put in a section above called "Setting up a domain".